<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>354</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>213</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCI通道设置</string>
  </property>
  <widget class="QComboBox" name="cmbDevice">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblDevice">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>设备：</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color:rgb(230, 240, 250);
    border-radius: 10px;
    border: 2px groove gray;
    border-style: outset;

    font:bold;
}

QPushButton:hover{
    background-color:white;
    color: black;
    }
QPushButton:pressed{
    background-color:rgb(85, 170, 255);
    border-style: inset;
    }</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color:rgb(230, 240, 250);
    border-radius: 10px;
    border: 2px groove gray;
    border-style: outset;

    font:bold;
}

QPushButton:hover{
    background-color:white;
    color: black;
    }
QPushButton:pressed{
    background-color:rgb(85, 170, 255);
    border-style: inset;
    }</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>AO设置</string>
   </property>
   <widget class="QComboBox" name="cmbChannelStart">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lblValueRange">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>84</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>输出最大值：</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblChannelCount">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>52</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>通道数量</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblChannelStart">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>起始通道：</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelStart</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbValueRange">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>84</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbChannelCount">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>52</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>高速AI采集配置</string>
   </property>
   <widget class="QLabel" name="lblChannelCount_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>56</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>通道数量：</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelCount</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtClockRatePerChan_AI">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbValueRange_AI">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lblValueRange_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>88</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>输入最大电压：</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblClockRate">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>采样频率</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbChannelStart_AI">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lblChannelStart_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>26</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>起始通道：</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelStart</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblRateUnit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>121</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbChannelCount_AI">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lblSectionLength">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>缓冲区长度：</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSectionLength_AI">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>位移采集</string>
   </property>
   <widget class="QRadioButton" name="radioButton_ENC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ENC_7480</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_SSI">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SSI采集卡</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cmbDevice</tabstop>
  <tabstop>cmbChannelStart</tabstop>
  <tabstop>cmbChannelCount</tabstop>
  <tabstop>cmbValueRange</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
